qa/integration/framework/fixture.rb

```diff
@@ -0,0 +1,81 @@
+require_relative "../services/service_locator"
+
+# A class that holds all fixtures for a given test file. This deals with
+# bootstrapping services, dealing with config files, inputs etc
+class Fixture
+  attr_reader :input
+  attr_reader :actual_output
+  attr_reader :test_dir
+
+  class TemplateContext
+    attr_reader :options
+
+    def initialize(options)
+      @options = options
+    end
+
+    def get_binding
+      binding
+    end
+  end
```

Comment:
Just a personal note here for other reviewers, I could have pass `self` to the ERB class, but I personally find it messy to give access to everything present in the class when it is not needed.